kitsune/kadmin/admin.py

```diff
@@ -49,10 +48,14 @@ def celery_settings(request):
 
 
 def env(request):
-    """Admin view that displays the wsgi env."""
-    return http.HttpResponse(u'<pre>%s</pre>' % (jinja2.escape(request)))
-
-admin.site.register_view('env', view=env, name='WSGI Environment')
+    """Admin view that displays env info."""
+    return render_to_response('kadmin/env_view.html', {
+        'request': request,
```

Comment:
These are django templates though. And now I am confused as to why it works on fjord and not here.